sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,5 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0018: Protocol method should have requestContext as the last parameter and don't have model as parameter type or return type. Protocol method should not have optional parameters if ambiguity exists between protocol method and convenience method.", Justification = "Flag keep-non-overloadable-protocol-signature is on.")]
```

Comment:
This should be inline. We should avoid global suppressions because they are non-descript. Please continue using the #pragmas.